rest/route/reliability.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 const (
-	reliabilityAPIMaxNumTasksLimit = 50
+	reliabilityAPIMaxNumTasksLimit = 1000
```

Comment:
can this have a comment to explain what this is and what the impact of changing it is?

Reply:
:+1: